### ISSUE TITLE
CI integration test failure: Docker link/alias

### ISSUE THREAD

**Issue by mhagen-vmware** [vmware/vic#2483]
From build:
https://ci.vmware.run/vmware/vic/5174

[integration_logs_5174_817e9b897eafaab0d8cbaecf3aa5d948771a5b9d.zip](https://github.com/vmware/vic/files/495965/integration_logs_5174_817e9b897eafaab0d8cbaecf3aa5d948771a5b9d.zip)

```
time="2016-09-27T14:40:55Z" level=error msg="attach connector: Connection not found error for id:40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216: context deadline exceeded" 
time="2016-09-27T14:40:55Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/attach.(*Connector).Get:75] [29.999137394s] 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
time="2016-09-27T14:40:55Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/attach.(*Server).Get:90] [29.99918297s] 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
time="2016-09-27T14:40:55Z" level=error msg="No stdout found for 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216: attach connector: Connection not found error for id:40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216: context deadline exceeded" 
time="2016-09-27T14:40:55Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*InteractionHandlersImpl).ContainerGetStdoutHandler:144] [29.999244394s] 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
time="2016-09-27T14:40:55Z" level=debug msg="attach connector:  Giving up on connection for 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
time="2016-09-27T14:40:55Z" level=info msg="attach connector:  Connection not found yet for 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
time="2016-09-27T14:40:55Z" level=debug msg="[BEGIN] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*ContainersHandlersImpl).ContainerWaitHandler:338] 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216:-1" 
time="2016-09-27T14:40:55Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*ContainersHandlersImpl).ContainerWaitHandler:338] [41.162µs] 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216:-1" 
time="2016-09-27T14:40:55Z" level=error msg="attach connector: Connection not found error for id:40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216: context deadline exceeded" 
time="2016-09-27T14:40:55Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/attach.(*Connector).Get:75] [30.007061757s] 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
time="2016-09-27T14:40:55Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/attach.(*Server).Get:90] [30.007083212s] 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
time="2016-09-27T14:40:55Z" level=error msg="No stderr found for 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216: attach connector: Connection not found error for id:40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216: context deadline exceeded" 
time="2016-09-27T14:40:55Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*InteractionHandlersImpl).ContainerGetStderrHandler:178] [30.007156464s] 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
time="2016-09-27T14:40:55Z" level=debug msg="attach connector:  Giving up on connection for 40e5b2da2d7a9773c8bf5ae952a56a3863d6a2fe285fab3ec199f1451083a216" 
```

**Comment by anchal-agrawal** [comment]
Updated the title to reflect the [failing integration test](https://github.com/vmware/vic/blob/master/tests/test-cases/Group1-Docker-Commands/1-24-Docker-Link.robot#L48). The failure is transient and network-related.

**Comment by anchal-agrawal** [comment]
A related failure is in the `Create linked containers that can ping` test from `1-4-Docker-Create`, as seen in https://ci.vmware.run/vmware/vic/5233.

```
Create linked containers that can ping                                | FAIL |
'PING busy1 (172.16.0.2): 56 data bytes
64 bytes from 172.16.0.2: seq=0 ttl=64 time=34.700 ms

--- busy1 ping statistics ---
2 packets transmitted, 1 packets received, 50% packet loss
round-trip min/avg/max = 34.700/34.700/34.700 ms' does not contain '2 packets transmitted, 2 packets received'
```

**Comment by anchal-agrawal** [comment]
Related to #2524.

**Comment by anchal-agrawal** [comment]
Closing as this was resolved in #2532. I'll reopen if we see this again.
